readthedocs/gold/views.py

```diff
@@ -52,6 +60,7 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         domains = Domain.objects.filter(project__users=self.request.user)
         context['domains'] = domains
+        context['stripe_publishable'] = settings.STRIPE_PUBLISHABLE
```

Comment:
Is there a reason we need this if we're POST'ing to our URL then redirecting? Can we not just have the key in our backend and pass it along?

Reply:
This was tricky to me as well. I wanted _our backend_ to redirect directly to Stripe Checkout, but for some reason, that has to be done in frontend via Javascript :shrug: 

https://stripe.com/docs/api/checkout/sessions/object#checkout_session_object-id

The official Python example does it in this way also, so I thought it was not possible to do it from the backend.

Docs: https://stripe.com/docs/billing/subscriptions/checkout

However, for the Stripe Customer Portal _it is_ possible to do the redirect from Python.

Reply:
Interesting.